src/tokens/evo-light.css

```diff
@@ -40,7 +40,9 @@
     --color-state-inverse-hover: #343434;
     --color-state-inverse-active: #323232;
     --color-state-accent-hover: #2854d9;
+    --color-state-button-accent-hover: #3461e9;
     --color-state-accent-active: #254fd2;
+    --color-state-button-accent-active: #3461e9;
```

Comment:
Is 'accent' the correct name. I thought 'accent' was for the blue accent background only.

Reply:
Not sure, I thought it was for blue in general
What would be the correct name then?

Reply:
Actually, DS should be providing the token name.

Reply:
@ianmcburnie , what do you think of these tokens? They avoid component-level name (`button`) and specific color variation name (`accent`)...

`````
--color-state-hover-secondary-stroke
--color-state-focus-secondary-stroke
--color-state-active-secondary-stroke

--color-state-hover-foreground-on-secondary
--color-state-focus-foreground-on-secondary
--color-state-active-foreground-on-secondary

And a similar pattern for destructive - 

--color-state-hover-foreground-on-secondary-destructive
...
--color-state-hover-secondary-destructive-stroke
...
`````

Reply:
Actually, I was just looking at primary button and seeing what we are doing there for token naming, and notice we don't use tokens, just a filter and blend: https://github.com/eBay/skin/blob/b796a6249e9e50c132d4ecb07baf91b2193d81d1/dist/button/button.css#L156

Can we not do something similar here?

Reply:
If not, then yes @ArtBlue those look good.